Formfield button

Cook-Self-Service opened the pull request.

nikosdion commented:
You are still including this line which I told you I disagree with:
```php
$onclick  = $this->onclick ? 'onclick="' . str_replace('[ITEM:ID]', $this->item->getId(), $this->onclick) . '"' : '';
```

Also, you need to update your repository before making a branch to create a pull request. Your PR is unmergeable as is.

Cook-Self-Service commented:
Ok, I have now removed it.
Personly I do not see any problem parsing [ITEM:ID] with the ID of the item. Anyways… I removed that.

About my repository, I do not understand how GitHub is working.
I removed all my branches.
I Sync in local.
I created new branch
Commit + Pull request.

I gives me the red icon that it cannot be merged automatically. What do I miss ?






Le 28 juil. 2015 à 07:38, Nicholas K. Dionysopoulos <notifications@github.com> a écrit :

> You are still including this line which I told you I disagree with:
> 
> $onclick  = $this->onclick ? 'onclick="' . str_replace('[ITEM:ID]', $this->item->getId(), $this->onclick) . '"' : '';
> Also, you need to update your repository before making a branch to create a pull request. Your PR is unmergeable as is.
> 
> —
> Reply to this email directly or view it on GitHub.
>
